Jenkinsfile

```diff
@@ -9,9 +9,11 @@ pipeline {
     }
     parameters {
         choice(name: "CHANNEL", choices: ["nightly", "dev", "beta", "release"], description: "")
+        choice(name: "BUILD_TYPE", choices: ["Release", "Debug"], description: "")
```

Comment:
Does this have a default value? When Jenkins is polling (and detects the change), how does the value get assigned? ex: I would imagine it defaults to `Release`